packages/generator-volto/generators/app/templates/package.json.tpl

```diff
@@ -142,7 +141,6 @@
     "eslint-plugin-prettier": "3.1.3",
     "jest-junit": "8.0.0",
     "mrs-developer": "*",
-    "postcss": "8.3.11",
```

Comment:
This is necessary for add-ons, if they have stylesheets, for current stylelint to work well.

Reply:
Yeah, I'm not sure what I was thinking here. Also I think razzle expects it to be a peer dependency, so needs to be specified at the top level.